libnopegl/src/text.c

```diff
@@ -318,6 +318,9 @@ static uint64_t prng64_next(uint64_t *state)
 
 static void shuffle(uint64_t *rng_state, size_t *positions, size_t n)
 {
+    if (n == 0)
+        return;
```

Comment:
0 segment is likely going to cause divisions by zero and all sort of problems. It might be possible to trigger it so I'll investigate, but please drop this commit since it should be an assert that deep in the codepath

Reply:
Up to you, dropped.